devicetypes/smartthings/aeon-home-energy-meter.src/aeon-home-energy-meter.groovy

```diff
@@ -198,4 +191,21 @@ private secEncap(physicalgraph.zwave.Command cmd) {
 
 private crcEncap(physicalgraph.zwave.Command cmd) {
 	zwave.crc16EncapV1.crc16Encap().encapsulate(cmd).format()
-}
\ No newline at end of file
+}
+
+private getVersions() {
+	if (zwaveInfo.model.equals("005F"))
+		[
+				0x32: 1,  // Meter
+				0x70: 1,  // Configuration
+				0x72: 1,  // ManufacturerSpecific
+		]
+	else
+		[
+				0x20: 1,  // Basic
+				0x32: 3,  // Meter
```

Comment:
Ah, there's no `zwaveEvent()` with a meterv3 object in its signature. The correct change is to either add that, or just change the parsing for all events to use meterv1.